src/Security/Authorization.cs

```diff
@@ -152,7 +159,14 @@ internal Authorization (IntPtr handle)
 			this.handle = handle;
 		}
 
-		[Deprecated (PlatformName.MacOSX, 10,7)]
+#if !NET
+		[Deprecated (PlatformName.MacOSX, 10, 7)]
+#else
+		[UnsupportedOSPlatform ("macos10.7")]
+#if MONOMAC
+		[Obsolete ("Starting with macos10.7 ", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
```

Comment:
same (not needed)

Reply:
Same message of the PInvoke documentation could be used.